lib/RuntimeLib.lua

```diff
@@ -61,7 +61,7 @@ local function isPlugin(object)
 end
 
 -- module resolution
-function TS.getModule(object, moduleName)
+function TS.getModule(object, moduleName, scope)
```

Comment:
This is a little funky from an emit perspective. But I think we can get around it while still maintaining backwards compatibility:
```Lua
function TS.getModule(object, scope, moduleName)
    if moduleName == nil then
        moduleName = scope
        scope = "@rbxts"
    end
    -- ...
end
```